internal/provider/resource_password.go

```diff
@@ -15,7 +17,17 @@ func (r resourcePasswordType) GetSchema(context.Context) (tfsdk.Schema, diag.Dia
 		"data handling in the [Terraform documentation](https://www.terraform.io/docs/language/state/sensitive-data.html).\n" +
 		"\n" +
 		"This resource *does* use a cryptographic random number generator."
-	return getStringSchemaV1(true, description), nil
+
+	schema := getStringSchemaV1(true, description)
```

Comment:
Nit: the name of the function `getStringSchemaV1` makes one thing that the schema that comes out of it is `V1`.

But it's not right? It's `V0`. `V1` is what you are creating now, by adding `bcrypt_hash`.

IMHO: I'd try to preserve consistency and have 2 functions able to provide the schema, and the version correctly respected in their name.

Reply:
Good point. Have refactored.